### DISCUSSION TITLE
Having a hard time finding any ressources (guides/tutorials)

### DISCUSSION THREAD

**Discussion by Thomas-Giaroli** [dokan-dev/dokany#1142] [Q&A] [answered]
Hey guys, I just began to work for a society and they prompted me with the task of creating a virtual reader for a distant server. I stumbled upon Dokany and it looks perfect for my mission but I have no experience whatsoever with Windows development and with Dokan/Dokany, so i'm having a hard time finding any good ressources or guides to help me. Do you guys have any starting point for me?

By the way i'm probably going to code this project in C++, is there anything i should know about Dokany usage with C++?

**Comment by Liryna** [comment, marked as answer]
Hi @Thomas-Giaroli ,

[Here](https://github.com/dokan-dev/dokany/blob/master/README.md) is a good start. You will find the [documentation](https://dokan-dev.github.io/dokany-doc/html/). If you need FS examples, you have the [samples](https://github.com/dokan-dev/dokany/tree/master/samples) with even `memfs` written in C++ which could interest you.

**Reply by Thomas-Giaroli** [reply]
Thanks a lot for the reply, i'll check all of this right away!

Have a good day
